Classes/Emogrifier.php

```diff
@@ -397,6 +420,57 @@ public function removeUnprocessableHtmlTag($tagName)
     }
 
     /**
+     * Normalizes the value of the "style" attribute and saves it.
+     *
+     * @param \DOMElement $node
+     *
+     * @return void
+     */
+    private function saveOriginalStyle(\DOMElement $node)
```

Comment:
Actually, this method does not really "save" the original styles to somewhere else. It just normalizes the contents of the style attributes and overwrites them in-place. So I'd opt for naming this method "normalizeStyleAttributes".